### DISCUSSION TITLE
How to prevent closing `Select` or `Combobox` Popover after choosing an item

### DISCUSSION THREAD

**Discussion by sadeghbarati** [adobe/react-spectrum#5730] [Q&A] [answered]
Something like RadixUI `event.preventDefault` is very useful or an example in the documentation :pray:

**Comment by reidbarber** [comment, marked as answer]
For this case, you can control the Select or Combobox's `isOpen` prop yourself.

**Reply by sadeghbarati** [reply]
Adding a page like RadixUI Migration would be so nice 💯
